examples/linear_extension/linear_extension.jl

```diff
@@ -115,10 +114,10 @@ function run_linear_extension_example()
     # -------------------------------
     ## Displacements
     "Computes displacements numeric solution uᵢ, uⱼ and uₖ for analytic validation."
-    function u_ijk_analytic(λᵥ::Vector{<:Real}, x₀::Real, y₀::Real, z₀::Real, ν::Real=ν, E::Real=E)
-        𝐶(t) = tension(t) * (1 - ν - 2ν^2) / (1 - ν)
+    function u_ijk_analytic(λᵥ::Vector{<:Real}, x₀::Real, y₀::Real, z₀::Real, nu::Real=nu, E::Real=E)
+        C(t) = tension(t) * (1 - nu - 2nu^2) / (1 - nu)
```

Comment:
Same comment; these should better use math notation imo.

Reply:
idem as the other conversation above